src/certificate/Certificate.tsx

```diff
@@ -254,9 +259,33 @@ export default function Certificate() {
                     <Col md={ 8 }>
                         <h2>Legal Officer Case</h2>
                         <h1>CERTIFICATE</h1>
-                        <IntroductionText loc={ loc } />
                     </Col>
                 </Row>
+                { tokenGated &&
+                    <Row>
+                        <Col md={ 5 }>
+                            <Authenticate
+                                walletType={ searchParams.get("wallet") }
+                                locId={ loc.data.id }
+                                item={ collectionItem! }
+                                owner={ loc.data.ownerAddress }
+                                setTokenForDownload={ setTokenForDownload }
+                            />
+                        </Col>
+                        <Col md={ 7 }>
+                            <IntroductionText loc={ loc } tokenGated={ true } />
+                        </Col>
+                    </Row>
+                }
+                { !tokenGated &&
+                    <Row>
+                        <Col md={ 2 }/>
+                        <Col md={ 8 }>
+                            <IntroductionText loc={ loc } tokenGated={ false } />
+                        </Col>
```

Comment:
This is a bit odd? What do you want to achieve exactly?

Reply:
Not sure to understand your question. If you refer to the attribute `tokenGated`, it's meant to adapt the introduction text when you are (or not) in a token-gated collection item certificate.

Reply:
I was referring to the fact that you are using a `Col` as a left margin, which is not a good practice.